runner/Cargo.toml

```
@@ -0,0 +1,40 @@
+[package]
+name = "runner"
+version = "0.1.0"
+edition = "2021"
+
+[features]
+default = ["json"]
+json = ["serde", "serde_json", "seccompiler/json"]
+
+[[bin]]
+name = "runner"
+path = "src/main.rs"
+
+[lib]
+features = ["json"]
+name = "runner"
+path = "lib/lib.rs"
+
+[[test]]
+features = ["json"]
+name = "test"
+path = "test/test.rs"
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+sharedlib = "7.0.0"
+libc = "0.2.1"
+seccompiler = {version = "0.2.0", features=["json"] }
+shh = "1.0.1"
+fork = "0.1.18"
+slog = "2.7.0"
+interprocess = "1.1.1"
+serde = {version = "1.0.130", features=["derive"], optional = true }
+serde_json = {version = "^1.0.9", optional = true}
+
+[dependencies.compiler]
+path = "../compiler"
+
+[build-dependencies]
+bindgen = "0.59.1"
```

```suggestion
bindgen = "0.59.1"
```suggestion
bindgen = "0.59.1"
```suggestion
bindgen = "0.59.1"
```
```
```